doc/getting-started.md

```diff
@@ -104,31 +104,10 @@ rm -rf node_modules
 npm install
 ```
 
-#### Prettier
+### Having trouble? 
 
-[Prettier](https://prettier.io/) is used to format this codebase.
-
-If you've forked TerriaMap prior to prettier being applied, merging will be a hassle. Here's how to do it (relatively) painlessly:
-
-* Create a new branch for your merge: `git checkout -b whatever-merge`
-* If you haven't already installed `npm-merge-driver`, add it to your local git config via `npx npm-merge-driver install`.
-* Merge the last commit before prettier into your branch: `git merge pre-prettier`. Resolve any conflicts as usual. If utilising yarn, it will resolve any `yarn-lock.json` conflicts automatically with another `yarn install`
-* Commit the merge above.
-* Merge the commit that ran prettier on all the source files into your branch. This is likely to cause heaps of conflicts, but because you've already merged the last commit before prettier, and because the prettier commit only changes formatting, you can safely accept your version any time there is a conflict. `git merge post-prettier --strategy=ours --no-commit`
-* Run prettier on the result of the merge before committing it: `npm run prettier`
-* Commit the merged result.
-* Merge master into your branch in order to pick up any changes in master that happened after the prettiergeddon: `git merge origin/master`. Resolve any conflicts as normal.
-* Commit and push your branch.
-* Open a pull request of your merge branch into the original one. It should merge cleanly.
-
-If you're merging a branch-that-has-already-followed-this-procedure into your branch, you can follow the procedure above except that you need to use different commits instead of `pre-prettier` and `post-prettier`. Look at the commits in the source branch and you should see two that look like this:
-
-<img src="../contributing/img/prettier-commits.png" />
-
-Use the commit hash of the commit that merged `pre-prettier` in place of `pre-prettier` above. Use the commit hash of the commit that merged `post-prettier` in place of `post-prettier` above.
-
-Having trouble? Drop by the TerriaJS [gitter](https://gitter.im/TerriaJS/terriajs) or [forum](https://groups.google.com/forum/#!forum/terriajs) and we'll be happy to help!
```

Comment:
Removed the section on prettier because the change happened long ago and it is now the default in `next` branch.